.rubocop.yml

```diff
@@ -38,6 +38,10 @@ Style/NumericLiteralPrefix:
 Style/TernaryParentheses:
   Enabled: false
 
+# sometimes it is usefull to have those empty methods
```

Comment:
I am in favor of having this check disabled. :+1: I don't find the one-line style to be major win.